packages/pigeon/README.md

```diff
@@ -8,24 +8,39 @@ host platform type-safe and easier.
 
 ## Supported Platforms
 
-Currently Pigeon only supports generating Objective-C code for usage on iOS and calling host functions from Flutter.
+Currently Pigeon only supports generating Objective-C code for usage on iOS and
+Java code for Android.
 
 ## Runtime Requirements
 
-Pigeon generates all the code that is needed to communicate between Flutter and the host platform, there is no extra runtime requirement.  A plugin author doesn't need to worry about conflicting versions of Pigeon.
+Pigeon generates all the code that is needed to communicate between Flutter and
+the host platform, there is no extra runtime requirement.  A plugin author
+doesn't need to worry about conflicting versions of Pigeon.
 
 ## Usage
 
-### Steps
+### Flutter calling into iOS Steps
 
 1) Add Pigeon as a dev_dependency.
 1) Make a ".dart" file outside of your "lib" directory for defining the communication interface.
 1) Run pigeon on your ".dart" file to generate the required Dart and Objective-C code.
-1) Add the generated code to your `ios/Runner.xcworkspace` XCode project for compilation.
+1) Add the generated Dart code to `lib` for compilation.
+1) Add the generated Objective-C code to your `ios/Runner.xcworkspace` XCode project for compilation.
```

Comment:
or just your normal Xcode project if they're doing add-to-app

Reply:
done